lib/server.js

```diff
@@ -149,11 +153,15 @@ class S3Server {
 
         // TODO this should wait for metadata healthcheck to be ok
         // TODO only do this in cluster master
-        setTimeout(() => {
-            if (enableRemoteManagement) {
-                initManagement(logger.newRequestLogger());
-            }
-        }, 5000);
+        if (!isManagementAgentUsed()) {
```

Comment:
The management agent client shouldn't be initialized either if we're disabling remote management altogether

Reply:
done